base/docs/helpdb/Base.jl

```diff
@@ -1767,16 +1767,19 @@ deserialize
 """
     length(collection) -> Integer
 
-For ordered, indexable collections, returns the maximum index `i` for which `getindex(collection, i)`
-is valid.
-For unordered collections, returns the number of elements.
+Return the number of elements in collection.
```

Comment:
``
`collection`
``
?

Actually, maybe change to `length(A) -> Integer` since the word collection is used to reference something else then the argument just below?